prototypes/prototype.adoc

```diff
@@ -0,0 +1,44 @@
+---
+authors: Han Solo <han@example.com>, Frodo Baggins <frodo@example.com>
+state: predraft
+---
+= RFD <NUMBER> <Title>
+:author: Han Solo <han@example.com>; Frodo Baggins <frodo@example.com>
```

Comment:
@trentm asciidoc does make use of this attribute somehow, somewhere.  If you look at the rendered HTML, it has a placeholder for this value:

<img width="951" alt="image" src="https://user-images.githubusercontent.com/349516/28794345-d7470506-75ea-11e7-9271-4209434a2d79.png">

The front-matter for jekyll is purely for Jekyll.  asciidoc ignores it completely (and though does save the value for reference, the content of the front-matter is un-indexable by asciidoc thereby rendering it effectively useless to asciidoc).  I'm indifferent to its presence despite the fact that it doesn't add any value to the meaning of the content and also doesn't improve Github's rendering of the document (compare GH's rendering of asciidoc vs the actual `asciidoctor`-rendered version at https://cdn.rawgit.com/joyent/rfd/1bc065db/rfd/0106/README.html).  I assumed the front-matter was being used somehow by one of our tools and was therefore a structurally important thing to leave in place for compatibility reasons.